fcrepo-kernel-api/src/main/java/org/fcrepo/kernel/api/models/FedoraResource.java

```diff
@@ -242,4 +242,11 @@ void replaceProperties(final IdentifierConverter<Resource, FedoraResource> idTra
      * @return the node for this object at the version provided
      */
     Node getNodeVersion(String label);
+
+    /**
+     * This method returns the version label of this frozen resource.
+     * If this resource is not frozen, null is returned.
+     * @return version label
+     */
+    String getVersionLabelOfFrozenResource();
```

Comment:
The concept of "FrozenResource" does leave a taste of JCR. The point of the method is to get the label of a version. I was keeping the terminology in line with the other methods on this interface:
- `isFrozenResource`: https://github.com/awoods/fcrepo4/blob/fcrepo-1942/fcrepo-kernel-api/src/main/java/org/fcrepo/kernel/api/models/FedoraResource.java#L225
- `getUnfrozenResource`: https://github.com/awoods/fcrepo4/blob/fcrepo-1942/fcrepo-kernel-api/src/main/java/org/fcrepo/kernel/api/models/FedoraResource.java#L237

..but would be happy to change the name if you have a suggestion.